Source/Core/Common/CMakeLists.txt

```diff
@@ -69,6 +69,14 @@ if (APPLE)
     ${COREFOUNDATION_LIBRARY}
     ${IOK_LIBRARY}
   )
+elseif(WIN32)
+  target_link_libraries(common
+  PRIVATE
+    kernel32
+    hlwapi
+    opengl32
+    winmm
```

Comment:
Should these use the `.lib` suffix, given it's not a target defined in our cmake scripts?

Reply:
It's not required, as it is compiling fine. But it's probably a good idea anyway, just to differentiate